gtubt/lib/service/authentication.dart

```diff
@@ -44,4 +44,8 @@ class AuthService {
   Future<auth.User> getUser() async {
     return _auth.currentUser;
   }
+
+   Future<void> sendPasswordResetEmail(String email) async {
+    return _auth.sendPasswordResetEmail(email: email);
```

Comment:
burada FirebaseAuthException exception gelebilir. bunun kodu `user-not-found` olabiliyor. bunu handle etmemiz gerekli diye dusunuyorum.

Reply:
Exception degil de return degerini kontrol etmemiz gerekir mi? Map fonksiyonunda exception handlelanmis zaten

Reply:
tamam orayi gormemiism. her turlu durum hamdle edilmiş. return degeri yok zaten ama bunu await ile beklenmesi gerekli @aErgani